.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 assignees: ""
 ---
 
-_Please note as part of [our Q2 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/6161) we paused adding new wallets until we establish new, expanded criteria._
+Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/). Only continue with the issue if your wallet meets the criteria listed there.
 
-Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/).
+If the wallet does meet our list policy, please complete the following information which we need to accurately list the wallet.
 
-Only continue with the issue if your wallet meets the criteria listed there.
+### Project info
 
-If it does complete the following information which we need to accurately list the wallet.
+**Wallet name**
 
-**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team or some other method.**
+  <!-- Please provide the official name of the wallet -->
+
+**Wallet description**
+
+<!-- Please provide a short 1-2 sentence description of the wallet without marketing claims. Avoid claims like “the best Ethereum wallet”. We will ask for revisions if this comes across as marketing-focused. -->
+
+**Wallet logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Background color for brand logo**
+
+<!-- Please provide a hex code for the brand color. This will be added to the background of the wallet card. -->
+
+**URL to the project**
+
+<!-- Please provide a URL (e.g. to the website of the wallet) -->
+
+**URL to the documentation**
```

Comment:
I think we can remove - seems to be a duplicate of the question in the support section:

> **What educational resources/documentation do you provide to users?**

Reply:
@corwintines Was the goal here to distinguish between _developer documentation_ for the wallet itself and _user education_ on how to use the software?

Reply:
I think both, but I can edit the copy for that later question to capture this.